content/shapes/functions.ts

```diff
@@ -2250,10 +3235,10 @@ class Slice {
     };
     this._bounds = new Polygon(...sliceSides.points);
     this._pivot = (new Segment((new Segment(pointA, pointB)).midpoint, circleCenter)).midpoint.rotate(-(arcAngle / 2), circleCenter).rotate(this.angle, circleCenter);
-    this.transAnim = null;
-    this.rotAnim = null;
-    this.rotTarget = null;
-    this.transTarget = null;
+    this.transAnim = undefined;
+    this.rotAnim = undefined;
+    this.rotTarget = undefined;
+    this.transTarget = undefined;
```

Comment:
See comment above: if you do that, you don't have to initialize any properties that are undefined.